### ISSUE TITLE
Does not save post context

### ISSUE THREAD

**Issue by abjugard** [xavdid/reddit-user-to-sqlite#8]
A comment is great, but it's hard to look back at what I posted without the context of the thread I replied to (really, the entire post should be saved, but only once in the case where I've made many comments in the same post).

Similarly, it's useful to be able to see replies to any posts I've backed up.

I realise doing this would result in a whole lot more requests and probably break the rules for API access. So perhaps also add code that ensures only 100 requests are made per minute?

**Comment by xavdid** [comment]
Yes, I'll have to think about this. I'm worried about having to make requests to store entire comment trees if I could help it. 

I could probably get away with getting the text of the immediate parent, if available. Would that be enough or would you want the comment pulled in as a full database row? What about the parent to that comment?

**Comment by wackget** [comment]
+1

For me I would ideally like to save the current comment *chain*, i.e. the equivalent of linking to a reddit comment with a parameter like `?context=9999999`.

**Comment by bheeshmpita** [comment]
Thanks @xavdid for this wonderful tool. 
I would like to get the entire post downloaded which i have interacted with (commented/upvoted/saved etc) if that is feasible.
